src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Basically, it should look something like this - 
```js
stopPropagation: function() {
...
if (this.isSimulated) return;
}
```

need a good test for it though